src/main/webapp/app/admin/metrics/metrics.component.html

```diff
@@ -10,7 +10,7 @@
         <ul class="list-group">
```

Comment:
this code seems to be used in many components. It should be made into a re usable component and then included everywhere